r/content/specify_data_types_and_schemas.Rmd

```diff
@@ -0,0 +1,205 @@
+# Defining Data Types
+
+As discussed in previous chapters, Arrow automatically handles the conversion of objects from native R data types to Arrow data types.  
+However, you might want to manually define data types, for example, to ensure interoperability with databases and data warehouse systems.
```

Comment:
Nit: This sentence doesn't quite work for me.  Even if you choose the data type Arrow will still need to convert from a native R type to an Arrow type.  Maybe something that uses the word "inference" or "picking the best data type".  For example:

```
Data in Arrow can be represented by a number of different data types.  When importing data from R the default behavior will pick the Arrow data type that is the safest match for the incoming R type.  However, you might want...
```

Reply:
Although even that example seems to fall short as description for the entire chapter because we should probably point out that this inference happens both when importing from R and when importing from files (or importing from any non-Arrow source really).

Reply:
Fair points, rephrased now.